### PR TITLE
Updated README.md.

### PR DESCRIPTION
**Pull request by Grubas7**
Changed readme. Before readme was basically copy-paste of same things for label, button, text view and text field.
Would be nice if someone who's native english would review this :)

### CODE REVIEW

**Comment by Grubas7** [comment]
I found few more useless white spaces, so please do not merge yet.

File: README.md [hunk 1]
```diff
@@ -17,19 +17,16 @@ Install with [CocoaPods](http://cocoapods.org). Add to your `Podfile`:
...
 ```
 
-## SSDynamicLabel
+## Example usage
 
-A label that responds when the user changes her preferred text size. Check out `Example` for a full example.
+You can create dynamic auto-sizing label, button, text field, and text view using a custom font and a base font size or a font descriptor. Check out full [example](https://github.com/splinesoft/SSDynamicText/blob/master/Example/SSDynamicTextExample/SSViewController.m).
 
-```objc
+UIKit views that responds when the user changes her preferred text size:
```
Comment (manicaesar) [inline README.md]: her -> his

File: README.md [hunk 2]
```diff
@@ -3,9 +3,9 @@ SSDynamicText
 
 [![Circle CI](https://circleci.com/gh/splinesoft/SSDynamicText.svg?style=svg)](https://circleci.com/gh/splinesoft/SSDynamicText) [![codecov.io](http://codecov.io/github/splinesoft/SSDynamicText/coverage.svg?branch=master)](http://codecov.io/github/splinesoft/SSDynamicText?branch=master)
  
-iOS 7's [`UIFontDescriptor`](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIFontDescriptor_Class/) is pretty neat. Also pretty neat is dynamic text that responds to the preferred text size that the user specified in Settings.app.
+iOS 7's `UIFontDescriptor` is pretty neat. Also pretty neat is dynamic text that responds to the preferred text size that the user specified in Settings.app.
```
Comment (manicaesar) [inline README.md]: Why did you remove the link?

File: README.md (hunk 1 above)
Comment (jhersh) [inline README.md]: her is fine.

File: README.md (hunk 2 above)
Comment (jhersh) [inline README.md]: Good catch, please restore this link.

File: README.md [hunk 3]
```diff
@@ -3,9 +3,9 @@ SSDynamicText
...
 
-What's not so neat, though, is that `+[UIFont preferredFontForTextStyle:]` only works with the system font, Helvetica Neue. What if you have custom fonts and want to respect the user's text size preference?
+What's not so neat, though, is that `+[UIFont preferredFontForTextStyle:]` only works with the system font, Helvetica Neue (iOS8) or San Francisco (iOS9). What if you have custom fonts and want to respect the user's text size preference?
```
Comment (jhersh) [inline README.md]: Let's add a space ("iOS 8" and "iOS 9")

File: README.md [hunk 4]
```diff
@@ -17,19 +17,16 @@ Install with [CocoaPods](http://cocoapods.org). Add to your `Podfile`:
...
-## SSDynamicLabel
+## Example usage
 
-A label that responds when the user changes her preferred text size. Check out `Example` for a full example.
+You can create dynamic auto-sizing label, button, text field, and text view using a custom font and a base font size or a font descriptor. Check out full [example](https://github.com/splinesoft/SSDynamicText/blob/master/Example/SSDynamicTextExample/SSViewController.m).
 
-```objc
+UIKit views that responds when the user changes her preferred text size:
+
+### SSDynamicLabel
```
Comment (jhersh) [inline README.md]: Should we use inline code style here, as with `NSAttributedString` below?

### `SSDynamicLabel`

File: README.md [hunk 5]
```diff
@@ -17,19 +17,16 @@ Install with [CocoaPods](http://cocoapods.org). Add to your `Podfile`:
 pod 'SSDynamicText', :head # YOLO
 ```
 
-## SSDynamicLabel
+## Example usage
 
-A label that responds when the user changes her preferred text size. Check out `Example` for a full example.
+You can create dynamic auto-sizing label, button, text field, and text view using a custom font and a base font size or a font descriptor. Check out full [example](https://github.com/splinesoft/SSDynamicText/blob/master/Example/SSDynamicTextExample/SSViewController.m).
```
Comment (jhersh) [inline README.md]:
```
`SSDynamicText` provides dynamic auto-sizing labels, buttons, text fields, and text views that respond when the user changes her preferred text size. Check out a full [example](...).
```

File: README.md (hunk 2 above)
Comment (Grubas7) [inline README.md]: I removed because below is a `UILabel`, `UIButton`, etc. and those objects have not link, so I though that everyone should have or no one.
But no problem. I'll restore link.

File: README.md (hunk 4 above)
Comment (Grubas7) [inline README.md]: This is a good idea from consist point of view, not so good from visual point of view ;)
I'll change.

File: README.md (hunk 5 above)
Comment (Grubas7) [inline README.md]: You right, much better.

Comment (Grubas7) [inline README.md]: Changed.

File: README.md (hunk 4 above)
Comment (Grubas7) [inline README.md]: Changed.

File: README.md (hunk 3 above)
Comment (Grubas7) [inline README.md]: Changed.

File: README.md (hunk 2 above)
Comment (Grubas7) [inline README.md]: Changed.

**Comment by Grubas7** [comment]
@jhersh ASAIK I can't squash this PR because of merge with splinesoft/master. Do you want new PR with one commit?

**Comment by jhersh** [comment]
@Grubas7 sure you can!

From your fork:

```bash
# one time only
git remote add upstream git@github.com:splinesoft/SSDynamicText.git

# switch to PR branch
git checkout master

# rebase on upstream
git pull --rebase upstream master

# reset/squash
git reset upstream/master

# create a single new commit
git add -A .
git commit -m "Updated README"

# force push back to fork, updating the PR
git push -f origin master
```

**Comment by Grubas7** [comment]
@jhersh Thanks a lot! 🎉

**Comment by jhersh** [comment]
Thanks. Feel free to merge after CI finishes :+1:

**Comment by jhersh** [comment]
Also, can we fix Circle's simulator to 9.0 as per #22?

**Comment by Grubas7** [comment]
Yes, this will be my next PR ;)
